client/lib/oauth2-clients.js

```diff
@@ -11,5 +11,5 @@ export const isCrowdsignalOAuth2Client = oauth2Client => {
 };
 
 export const isWooOAuth2Client = oauth2Client => {
-	return includes( [ 50019, 50915, 50916 ], oauth2Client.id );
+	return oauth2Client && includes( [ 50019, 50915, 50916 ], oauth2Client.id );
```

Comment:
Same comment from the WPCOM side - a comment explaining which IDs map to what clients would be helpful (if that can be in a public repo).